cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,37 @@
-describe('', () => {
-  before(() => {
+let user;
+let article;
 
+describe('Conduit article flow', () => {
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
+  it('Should create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
 
+    cy.findByPlaceholder('Article Title').type(article.title);
+    cy.findByPlaceholder('What\'s this').type(article.description);
+    cy.findByPlaceholder('Write your').type(article.body);
+
+    cy.get('[type="button"]').click();
```

Comment:
This line of code is not very clear. It would be better to use a more specific selector or add a data-cy attribute to the button and use cy.get('[data-cy=submit]').click() instead.